src/platform_impl/linux/x11/window.rs

```diff
@@ -585,6 +585,16 @@ impl UnownedWindow {
 
     fn set_fullscreen_inner(&self, fullscreen: Option<Fullscreen>) -> Option<util::Flusher<'_>> {
         let mut shared_state_lock = self.shared_state.lock();
+        // Map Borderless(`None`) to current monitor.
```

Comment:
```suggestion
        let mut shared_state_lock = self.shared_state.lock();
        
        // Map Borderless(`None`) to current monitor.
```

I'd just give it a touch of space.